NewDawn/NewDawn/ViewControllers/SettingPageViewControllers/EditProfileViewController.swift

```diff
@@ -67,27 +67,60 @@ class EditProfileTabelViewController: UITableViewController{
     @objc func back(sender: UIBarButtonItem) {
         // Get information from local and send to server
         // Go back to the root ViewController
-        self.dismiss(animated: true, completion: {})
-        self.navigationController?.popViewController(animated: true)
+        let request = EditProfileUtil.createRegistrationRequest()
+        
+        if request == nil {
+            return
+        }
+        let activityIndicator = self.prepareActivityIndicator()
+        self.processSessionTasks(request: request!){
+            register_response in
+            if let images = ImageUtil.getPersonalImagesWithData(), images.count != 0{
+                var images_count = 0
```

Comment:
Not sure why we increment this number within the loop, instead of directly use images.count after the for loop.

Reply:
Same as above